base/src/test/resources/failure/patterns/issues2/issue250.aya.txt

```diff
@@ -0,0 +1,11 @@
+In file $FILE:3:4 ->
+
+  1 | open data Crash
+  2 | | go Crash {
+  3 |   | ready => ready
+          ^---^
+
+Error: Attempting to use a definition `ready` which is not yet typechecked
```

Comment:
We need to generate tyck order of ctors   🐘 🐐

Reply:
I have a branch for that, it's called future

Reply:
How to deal with mutual ctors? Do you have any idea?

Reply:
Basically, data signatures first, function signatures second, data body third, function bodies last